### PR TITLE
Save user answer to db

### PR DESCRIPTION
**Pull request by mkayad**
### Jira link

[FPO-383](https://transformuk.atlassian.net/browse/FPO-383)

### What?

I have added/removed/altered:

- [x] Added save answer to db
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

- user answer needs to collected

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: src/controllers/identifyController.ts [hunk 1]
```diff
@@ -10,14 +11,41 @@ export class IdentifyController {
...
 
     res.render('identify', { goodsNomenclature })
   }
+
+  public async create (req: Request, res: Response): Promise<void> {
+    const session = req.session ?? {}
+    console.log(session.goodsNomenclature)
+    if (req.body.correctCNCode === 'yes' || req.body.correctCNCode === 'maybe') {
+      // save the answer
+      await Identification.create({
```
Comment (willfish) [inline src/controllers/identifyController.ts:32]: I think you still want to save the answer even in the `no` scenario?

Comment (mkayad) [reply src/controllers/identifyController.ts:32]: good question: I thought the no answer will be saved in the no scenario

Comment (mkayad) [reply src/controllers/identifyController.ts]: Changed the code to save for no answer as well

File: src/controllers/identifyController.ts [hunk 2]
```diff
@@ -10,14 +11,37 @@ export class IdentifyController {
...
 
-  public async show (_req: Request, res: Response): Promise<void> {
+  public async show (req: Request, res: Response): Promise<void> {
     if (this.sampler === null) {
       this.sampler = await DescriptionSampler.build()
     }
 
     const description: Description = this.sampler.sample()
     const goodsNomenclature = await this.client.get(description)
+    const session = req.session ?? {}
```
Comment (willfish) [inline src/controllers/identifyController.ts:21]: Ah we can just use hidden fields to include this in the POST request?

**Review by willfish — APPROVED** [review]

**Pull request merged**